cmd/cdi-operator/BUILD.bazel

```diff
@@ -9,10 +9,10 @@ go_library(
         "//pkg/operator/controller:go_default_library",
         "//pkg/util:go_default_library",
         "//staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1:go_default_library",
-        "//vendor/github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1:go_default_library",
         "//vendor/github.com/openshift/api/config/v1:go_default_library",
         "//vendor/github.com/openshift/api/route/v1:go_default_library",
         "//vendor/github.com/openshift/api/security/v1:go_default_library",
+        "//vendor/github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1:go_default_library",
```

Comment:
what exactly is a difference between github.com/prometheus-operator and github.com/coreos ?

Reply:
There is no difference. The URL "github.com/coreos/prometheus-operator" redirects to "github.com/prometheus-operator/prometheus-operator".